numpy/core/src/multiarray/compiled_base.c

```diff
@@ -112,20 +113,30 @@ arr_bincount(PyObject *NPY_UNUSED(self), PyObject *args, PyObject *kwds)
     }
     len = PyArray_SIZE(lst);
 
+    /*
+     * This if/else if can be removed by changing the argspec to O|On above,
+     * once we retire the deprecation
+     */
     if (mlength == Py_None) {
-        minlength = 0;
```

Comment:
Nope, because I set this to 0 in the declaration (like we would need to do with a `O|On` signature)